challenges/10exercises.js

```diff
@@ -1,26 +1,181 @@
-// Define a function max() that takes two numbers as arguments and returns the largest of them.
-// Use the if-then-else construct available in Javascript.
+// // 1. Define a function max() that takes two numbers as arguments and returns the largest of them.
+function max(number1, number2) {
+  if (number1 > number2) {
+    return number1;
+  }
+  if (number2 > number1) {
+    return number2;
+  }
+  if (number1 === number2) {
+    return "The two numbers are equal."
+  }
+}
+// // Use the if-then-else construct available in Javascript. ??(Wouldn't work for equal)
+//
+// // 2. Define a function maxOfThree() that takes three numbers as arguments and returns the largest of them.
+  function maxOfThree(number1, number2, number3) {
+    return Math.max(number1, number2, number3);
+  }
+//
+// // 3. Write a function that takes a character (i.e. a string of length 1) and returns true if it is a vowel, false otherwise.
+function isVowel(character) {
+  var vowels = ['a','e','i', 'o', 'u']
 
-// Define a function maxOfThree() that takes three numbers as arguments and returns the largest of them.
+  if (vowels.indexOf(character) !== -1){
+    return true;
+  } else {
+    return false;
+  }
```

Comment:
This is a good solution. Another approach could be to use regular expressions.
Here is a good site to test those:

http://regexr.com/

Here is a solution using regular expressions:
```
 return character.match(/([aeiou])/ig) !== null